tests: Re-enable live upgrade tests

rbradford opened the pull request:
And bump release verion used to v39.0

Fixes: #6134

Signed-off-by: Rob Bradford <rbradford@rivosinc.com>

rbradford committed:
tests: Re-enable live upgrade tests

And bump release verion used to v39.0

Fixes: #6134

Signed-off-by: Rob Bradford <rbradford@rivosinc.com>

likebreath commented:
I believe the test failures of live-upgrades are due to the smaller Azure VMs we are using for our CI workers (likely be OOMed). We can either 1) run some of the large tests sequentially or 2) reduce their guest memory size for live upgrades/migration tests.

rbradford committed:
tests: Serialise the live migration/upgrade NUMA/balloon tests

These tests use relatively large memory allocations and if they are
allowed to run in parallel can result in the container encountering an
OOM situation.

Signed-off-by: Rob Bradford <rbradford@rivosinc.com>

rbradford commented:
> I believe the test failures of live-upgrades are due to the smaller Azure VMs we are using for our CI workers (likely be OOMed). We can either 1) run some of the large tests sequentially or 2) reduce their guest memory size for live upgrades/migration tests.

It is interesting how they only failed on the AMD workers though..
